### DISCUSSION TITLE
iTag - iBeacon - Tasmota 11

### DISCUSSION THREAD

**Discussion by ceinmart** [arendst/Tasmota#20404] [Q&A]
Hi , 
I have a tasmota 11.1 installed on ESP32 DevKit. 

Currently, I use it to capture the data from two Xiaomi BLE temperature , using Blerry scripts. All works fines. 

Now I get two iTag (the cheaper) and trying to setup a ble tracker for them . 
The iTag macaddress appear on Bluetooth scan , but that is it... 
They don't log the ibeacon . 
I've activated the : ibeacon 1 : 

```
14:45:13.419 CMD: ibeacon
14:45:13.434 MQT: stat/esp32_ble_tracker1/RESULT = {"IBEACON":1}
```

But still not get nothing about IBEACON on the logs... 
The mac appear on : `tele/esp32_ble_tracker1/BLE = {"Time":"2024-01-05T14:43:20","BLEDevices":...`

I saw on the documentation that ibeacon isn't included into any pre-compiled , need to include:
`#define USE_IBEACON_ESP32 `
That is the reason?
I saw here other users using the ibeacon command successfully and didn't mention they needed to recompile the firmware manually.

![image](https://github.com/arendst/Tasmota/assets/7062410/eecef1dc-69db-445a-9935-a11bcb0dd85a)

**Comment by sfromis** [comment]
I've got beacons working with `BLEAddrFilter 3` and standard precompiled binary, latest version.

And less "noise" with `iBeaconOnlyAliased 1` after having assigned aliases to what I'm interested in seeing.

**Comment by ceinmart** [comment]
Hi @sfromis  , 

I saw your message about this command another topic , tried here , without success... 
Unfortunately, I need to stay on version 11.1, because the blerry scripts aren't stable on newer versions, the tasmota crashes constantly .

Not sure if I'm stuck with this problem because of tasmota version. 
I bought other ESP32 , I will try both versions, 11.1 and last release, see if this is the problem...  

Have you been able to use the itag button ?  
like people done using EspHOME.

My main objective here is use this iTag as scene button.

**Reply by sfromis** [reply]
My test has been with another beacon. And a config including doing a `SetOption115 1` to enable ESP32 MI32 BLE. Using `tasmota32-bluetooth.bin`
After `iBeacon 1` and `BLEAddrFilter 3`, the messages come rolling. My beacon is not battery-operated, and 10-15 seconds between messages from that MAC.

If you have stability issues with Blerry, did you mention that to the developer?
